cmake/dependencies/boost/CMakeLists.txt.in

```diff
@@ -21,36 +21,38 @@ ExternalProject_Add(
 
 	GIT_REPOSITORY "https://github.com/boostorg/boost.git"
 	GIT_TAG "boost-1.79.0"
-	GIT_SUBMODULES	tools/boost_install
-					tools/build
+	GIT_SUBMODULES	
+					libs/algorithm
+					libs/assert
+					libs/bind
 					libs/config
-					libs/headers
-					libs/predef
-					libs/test
-					libs/type_traits
-					libs/function
-					libs/smart_ptr
-					libs/preprocessor
+					libs/container_hash
 					libs/core
-					libs/throw_exception
-					libs/assert
-					libs/utility
-					libs/integer
 					libs/detail
-					libs/static_assert
-					libs/move
-					libs/type_index
-					libs/container_hash
-					libs/bind
+					libs/exception
+					libs/function
+					libs/headers
+					libs/integer
 					libs/io
 					libs/iterator
+					libs/move
 					libs/mpl
-					libs/algorithm
-					libs/exception
 					libs/numeric/conversion
+					libs/optional
+					libs/predef
+					libs/preprocessor
+					libs/program_options
```

Comment:
Only this was added. The rest was sorted, making this change hard to read. Sorry about that